introc/common_c_functions.tex

```diff
@@ -90,8 +95,7 @@ \subsection{stdin oriented functions}
 char *result = fgets(buffer, sizeof(buffer), stdin);
 \end{lstlisting}
 
-The result is NULL if there was an error or the end of the file is reached.
-Note, unlike \keyword{gets}, \keyword{fgets} copies the newline into the buffer, which you may want to discard.
+Note that, unlike \keyword{gets}, \keyword{fgets} copies the newline into the buffer, which may be discarded.
```

Comment:
The last phrase ("which may be discarded") is confusing. Perhaps we should leave it out -- people can decide for themselves if it helps them or not in their specific situation.